exporters/otlp/otlp_exporter_benchmark.cc

```diff
@@ -0,0 +1,124 @@
+#include "otlp_exporter.h"
+
+#include <benchmark/benchmark.h>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace otlp
+{
+
+const trace::TraceId kTraceId(std::array<const uint8_t, trace::TraceId::kSize>(
+    {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}));
+const trace::SpanId kSpanId(std::array<const uint8_t, trace::SpanId::kSize>(
+    {0, 0, 0, 0, 0, 0, 0, 2}));
+const trace::SpanId kParentSpanId(std::array<const uint8_t, trace::SpanId::kSize>(
+    {0, 0, 0, 0, 0, 0, 0, 3}));
+
+// OtlpExporterTestPeer is a friend class of OtlpExporter
+class OtlpExporterTestPeer
+{
+public:
+  std::unique_ptr<sdk::trace::SpanExporter> GetExporter(
+      std::unique_ptr<proto::collector::trace::v1::TraceService::StubInterface> &stub_interface)
+  {
+    return std::unique_ptr<sdk::trace::SpanExporter>(
+        new exporter::otlp::OtlpExporter(std::move(stub_interface)));
+  }
+};
+
+class FakeServiceStub : public proto::collector::trace::v1::TraceService::StubInterface
+{
+  grpc::Status Export(grpc::ClientContext *context,
+                      const proto::collector::trace::v1::ExportTraceServiceRequest &request,
+                      proto::collector::trace::v1::ExportTraceServiceResponse *response) override
+  {
+    return grpc::Status::OK;
+  }
+
+  grpc::ClientAsyncResponseReaderInterface<proto::collector::trace::v1::ExportTraceServiceResponse>
+      *AsyncExportRaw(grpc::ClientContext *context,
+                      const proto::collector::trace::v1::ExportTraceServiceRequest &request,
+                      grpc::CompletionQueue *cq) override
+  {
+    return NULL;
+  }
+
+  grpc::ClientAsyncResponseReaderInterface<proto::collector::trace::v1::ExportTraceServiceResponse>
+      *PrepareAsyncExportRaw(grpc::ClientContext *context,
+                             const proto::collector::trace::v1::ExportTraceServiceRequest &request,
+                             grpc::CompletionQueue *cq) override
+  {
+    return NULL;
+  }
+};
+
+// Benchmark Export() with span fields filled
+void BM_OtlpExporterFullSpans(benchmark::State &state)
+{
+  std::unique_ptr<OtlpExporterTestPeer> testpeer(new OtlpExporterTestPeer());
+
+  auto mock_stub = new FakeServiceStub();
+  std::unique_ptr<proto::collector::trace::v1::TraceService::StubInterface> stub_interface(
+      mock_stub);
+
+  auto exporter = testpeer->GetExporter(stub_interface);
+
+  const int kBatchSize = 200;
```

Comment:
Please make this a global, like [here](https://github.com/open-telemetry/opentelemetry-cpp/blob/master/sdk/test/common/circular_buffer_benchmark.cc#L18). So there's no danger of using different values for both benchmark tests.

Reply:
I added it to both tests to have more fine-grained control, but you're right that it's not really necessary!